stroeer/article/v1/service.proto

```diff
@@ -10,9 +10,34 @@ option java_package = "de.stroeer.article.v1";
 
 service ArticleService {
   rpc GetArticle (GetArticleRequest) returns (stroeer.core.v1.Article) {}
+
+  // Returns an Article page including all (SEO) relevant information for
+  // rendering (canonical web or AMP) an Article with the given id.
+  rpc GetArticlePage (GetArticlePageRequest) returns (ArticlePage) {}
 }
 
 message GetArticleRequest {
   string id = 1;
 }
 
+message GetArticlePageRequest {
+  string id = 1;
```

Comment:
what's the `id` in this case? A path like `/` or `/culture/`?

Reply:
the CMS id